fava/core/budgets.py

```diff
@@ -61,6 +61,19 @@ def parse_budgets(custom_entries):
                                 entry.values[1].value,
                                 entry.values[2].value.number,
                                 entry.values[2].value.currency)
+                if budget.period not in [
+                    "daily", "weekly", "monthly", "quarterly", "yearly"
+                ]:
+                    errors.append(
+                        BudgetError(
+                            entry.meta,
+                            'invalid budget period "%s", please use "daily", \
+                        "weekly", "monthly", "quarterly" or "yearly"'
```

Comment:
Should we put the possible values in the error message?

Reply:
I think it's the most user friendly option, while being clearly redundant.